GlotovES/HM3/task_1.py

```diff
@@ -0,0 +1,28 @@
+#####################################
+
+str_37 = 'If the implementation is easy to explain, it may be a good idea'
```

Comment:
nice job, but you should have used the full python philosophy text

Reply:
Is it necessary to finish?

Reply:
that would be a good finish this task
and also read carefully about the methods find() and count()